README.md

```diff
@@ -71,6 +71,12 @@ This driver supports Kerberos authentication by setting up the Kerberos fields i
 
 Please refer to the [Coordinator Kerberos Authentication](https://prestodb.io/docs/current/security/server.html) for server-side configuration.
 
+#### JWT authentication
+
+This driver supports JWT authentication by setting the `AccessToken` field in the configuration. Simply add the query parameter with the JWT bearer token to be used for authentication. This token will then be sent as a bearer token for all HTTP requests.
```

Comment:
```suggestion
This driver supports JWT authentication by setting the `AccessToken` field in the configuration. Add the query parameter with the JWT bearer token to be used for authentication. This token will then be sent as a bearer token for all HTTP requests.
```
(This is a big answer for a small change because I want to explain why.)

Technical documentation should not describe something as "simple" or "easy" because "simple" and "easy" are different for different users. For context, see Gitlab documentation style guide topic [Building Trust](https://docs.gitlab.com/ee/development/documentation/styleguide/#building-trust), and [Don’t say “simply” in your documentation by Jim Fisher](https://blog.knowledgeowl.com/blog/posts/dont-say-simply-jim-fisher/).

Reply:
Thanks for the valuable feedback! It's a great suggestion and looking back definitely something to keep in mind from a technical documentation perspective. I'll definitely keep it in mind for future contributions!